app/styles/ui/drag-elements/_cherry-pick-commit.scss

```diff
@@ -16,9 +16,11 @@
     flex-direction: row;
     align-items: center;
     background-color: var(--background-color);
+    overflow: hidden;
 
     .commit {
       border-bottom: none;
+      max-width: 300px;
```

Comment:
Not related to this PR: Just noticed that the commit ghost will expand to show entire commit message which is weird looking for really long commit messages.